cartographer_grpc/framework/service.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 The Cartographer Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CARTOGRAPHER_GRPC_FRAMEWORK_SERVICE_H
+#define CARTOGRAPHER_GRPC_FRAMEWORK_SERVICE_H
+
+#include <grpc++/impl/codegen/service_type.h>
```

Comment:
Shouldn't this be `#include "grpc++/impl/codegen/service_type.h"` together with the other include. Or why not?

Reply:
I was separating library from our internal headers.

Reply:
Consistency in our code base would be double quotes and in the same block as our headers for non-system libraries.

Reply:
Done.